src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[];
+  onSelect?: (person: Person) => void;
+};
+
+export const Autocomplete: React.FC<Props> = React.memo(
+  ({ people, onSelect = () => {} }) => {
+    return (
+      <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+        <div className="dropdown-content">
+          {people.map((person, index) => (
+            <div
+              className="dropdown-item"
+              data-cy="suggestion-item"
+              key={index}
+              onClick={() => onSelect(person)}
```

Comment:
see screencast. I can't to choose any person from the list

https://github.com/user-attachments/assets/204961fc-5062-4307-a4a2-7c99641dd938

```suggestion
              onMouseDown={() => onSelect(person)}
```

it can help to solve problem. (onMouseDown event executes before onBlur, onClick after onBlur)